0-ad-history-mod-package/simulation/templates/units/han/support_healer_b.xml

```diff
@@ -5,6 +5,10 @@
     <SelectionGroupName>units/han/support_healer_b</SelectionGroupName>
     <SpecificName>Yīshī</SpecificName>
     <Icon>units/han/support_healer.png</Icon>
+    <History>
+        Probably one of the most important Chinese physicians of all time was Zhan Zhongjin. He lived in the second and third centuries BC, the very last years of the Han Dynasty. Although we only know very little about his life, Zhan Zhongjin recorded his research and findings in a book with the title translating to “Treatise of Febrile and Other Diseases”. This book forms a crucial cornerstone of traditional Chinese medicine; he is considered the sage of Chinese medicine.
+        Similar to other revolutionary physicians in history, Zhang Zhongjin regarded the practice of medicine as a divine gift and something requiring high responsibility and conscientiousness. He documented the symptoms of his patients, the treatment, and the results very precisely. Because he witnessed the devastating epidemics that, among other things, caused the collapse of the Han Empire, his work mostly focuses on different kinds of infectious diseases, like fevers, and suggests medication, various herbs and other remedies.
```

Comment:
```suggestion
        Similar to other revolutionary physicians in history, Zhang Zhongjin regarded the practice of medicine as a divine gift and something requiring high responsibility and conscientiousness. He documented the symptoms of his patients, the treatment, and the results precisely. Because he witnessed the devastating epidemics that, among other things, caused the collapse of the Han Empire, his work mostly focuses on different kinds of infectious diseases, like fevers, and suggests medication, such as various herbs and other remedies.
```

Very precisely is redundant. We can lose the very.